src/runtime/storageNG/direct-store-muxer.ts

```diff
@@ -19,8 +19,9 @@ import {noAwait} from '../util.js';
 import {PropagatedException, reportSystemException} from '../arc-exceptions.js';
 import {ChannelConstructor} from '../channel-constructor.js';
 import {Identified, CRDTEntityTypeRecord} from '../crdt/crdt-entity.js';
+import {BiMap} from '../bimap.js';
 
-export type StoreRecord<T extends CRDTTypeRecord> = {type: 'record', store: DirectStore<T>, id: number} | {type: 'pending', promise: Promise<{type: 'record', store: DirectStore<T>, id: number}>};
+export type StoreRecord<T extends CRDTTypeRecord> = {type: 'record', store: DirectStore<T>, idMap: BiMap<number, number|Promise<number>>} | {type: 'pending', promise: Promise<{type: 'record', store: DirectStore<T>, idMap: BiMap<number, number|Promise<number>>}>};
```

Comment:
Add a comment describing what get's stored in the record (particularly what's in the bimap)